main.go

```diff
@@ -41,8 +41,8 @@ import (
 	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
 	loggingreconciler "github.com/giantswarm/logging-operator/pkg/logging-reconciler"
 	"github.com/giantswarm/logging-operator/pkg/reconciler"
-	grafanaagentconfig "github.com/giantswarm/logging-operator/pkg/resource/grafana-agent-config"
-	grafanaagentsecret "github.com/giantswarm/logging-operator/pkg/resource/grafana-agent-secret"
+	eventsloggerconfig "github.com/giantswarm/logging-operator/pkg/resource/grafana-agent-config"
```

Comment:
I think you can rename those in the same PR but this is reviewable for sure :)

Reply:
I'd rather keep this change as it is for now as renaming the imports themselves would lead to errors.